### PR TITLE
Personal dz60 keymap

### PR DESCRIPTION
**Pull request by pevecyan**

### CODE REVIEW

File: keyboards/dz60/keymaps/pevecyan/keymap.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+	LAYOUT(
+		KC_GRV, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_TRNS, KC_DEL,
+		KC_TRNS, RGB_TOG, RGB_MOD, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, RESET,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, BL_DEC, BL_TOGG, BL_INC, BL_STEP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+	LAYOUT(
+		KC_TRNS, M(1), M(2), M(3), M(4), M(5), M(6), M(7), M(8), M(9), M(10), M(11), M(12), KC_TRNS, KC_TRNS,
```
Comment (drashna) [inline keyboards/dz60/keymaps/pevecyan/keymap.c]: These `M()` keycodes aren't doing anything and should be replaced. 

```suggestion
		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
```

**Comment by pevecyan** [comment]
@vomindoraan Thanks for suggestions!
About your suggestion to merge layers, the previous version wasn't exactly correct. I want to achieve that if I hold CARON then I can write ČŽŠ with usual CZS and if I press then it sends ^. 
I am not sure if this is the correct way to achieve this.
Thanks for help

**Comment by drashna** [comment]
Awesome, thanks!

**Comment by vomindoraan** [comment]
@pevecyan Yes, `LT(3, SI_CIRC)` is indeed the way to do it. I have a similar setup. In case you find that you're typing Č, Š and Ž too quickly and are getting ^C, ^S and ^Z instead, try adding [`#define PERMISSIVE_HOLD`](https://docs.qmk.fm/#/feature_advanced_keycodes?id=permissive-hold) to your `config.h`.

**Pull request merged**